setup.py

```diff
@@ -522,7 +522,11 @@ def make_release_tree(self, base_dir, files):
             self.run_command('write_salt_ssh_packaging_file')
             self.filelist.files.append(os.path.basename(PACKAGED_FOR_SALT_SSH_FILE))
 
-        sdist.make_release_tree(self, base_dir, files)
+        def decode_files(files):
+            PY3 = sys.version_info[0] == 3
```

Comment:
Better reverse the logic, if py2, because when py4 comes out this will break. Makes sense?

Reply:
that is a solid point

Reply:
actually, it turnes out that IS_PY3 is already a thing in setup.py, so I am going to use that instead.

It just checks if the version is > (3,) so it should still work with py4